start.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+my_dir="$(dirname "$0")"
+
+xmlstarlet ed --inplace \
+	--update	"//Context/@path" --value $CONF_PATH \
```

Comment:
We should implement support for proxy configuration as well besides context path ie. `proxyName`, `proxyPort`, `scheme`, the latter of which probably should include some simple inference by `proxyPort`.